### DISCUSSION TITLE
Contributing upgrade of Maven Wrapper (Issue #3566)

### DISCUSSION THREAD

**Discussion by benkelaar** [renovatebot/renovate#14512] [Suggest an Idea]
Good morning!

[@breun](https://github.com/breun) strong-armed us into contributing [this feature](https://github.com/renovatebot/renovate/issues/3566), so [@trystian1](https://github.com/trystian1) and I are going to give it a go!

Besides what's written in the [contributing.md](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md) is there anything else we should be aware of?

Kind regards, Trystian & Bart

**Comment by rarkins** [comment]
Great!

Hi @benkelaar I hope that our contributing.md covers it. If in doubt, check out some other live PRs, if still any questions then feel free to follow up here. If you comment in that actual issue then GitHub will allow me to assign you to the issue so others can see
